docs/management/managing-tags.asciidoc

```diff
@@ -0,0 +1,103 @@
+[role="xpack"]
+[[managing-tags]]
+== Tags
+
+The *tags* management section allows you to manage your tags, and assign them
+to existing saved objects.
+
+Assigning tags to objects helps to categorize them, and allows to easily find
+the objects by filtering by tag(s) in the <<managing-saved-objects, Saved Objects management section>>
+or from the other applications.
+
+To access the tags management page, open the main menu, then click on *Stack Management > Tags*.
+
+[role="screenshot"]
+image::images/tags/tag-management-section.png[Tags management section]
+
+[float]
+=== Required permissions
+
+The `Tag Management` {kib} privilege is required to access the *Tags* management section.
+
+To add the privilege, open the menu, then click *Stack Management > Roles*.
+
+Having the `read` privilege allows a user to:
+- Access the `tags` management section
+- Assign tags to saved objects he has `write` permission for
+
+Having the `write` privilege allows a user to additionally:
+- Create tags
+- Delete tags
+- Edit existing tags
+
+NOTE: having the `Tag Management` {kib} privilege is not required to
+see tags assigned on objects the user can see, or to filter objects by tags
+in {kib} applications or from the navigational search.
+
+NOTE: having `write` `Tag Management` privilege is not sufficient
+to assign tags to saved objects. The user also needs to have `write` permission
+on the objects he wants to assign tags to.
```

Comment:
Let me know if this isn't accurate, but I think this is more specific to what we mean

```suggestion
NOTE: having `write` `Tag Management` privilege does not affect which saved objects
the user can assign tags to. The user needs to have `write` permission
on the objects they want to assign tags to.
```